src/datepicker/datepicker.component.ts

```diff
@@ -226,6 +226,16 @@ export class DatePicker implements
 
 	@Input() plugins = [];
 
+	/**
+	 * The minimum date that a user can start picking from.
+	 */
+	@Input() minDate: string | number;
+
+	/**
+	 * The maximum date that a user can pick to.
+	 */
+	@Input() maxDate: string | number;
+
```

Comment:
Hey @rkkp1023, this was intentionally excluded. At the time of the component creation, it was decided that we weren't going to create explicit wrapper inputs due to the possibility of changes upstream (flatpickr). If we start adding flatpickrOptions as inputs, we will then need to add ALL.

Instead, please pass the maxDate/minDate as part of the flatpickrOptions.

Reply:
Thanks, @Akshat55, Added flatpickerOptions in the storybook

Reply:
We tend to follow React implementation for the most part, however there are certain components that we decided to not 100% follow in terms of implementation. Date picker is one of those components. 

Date picker is dependent on a third-party library (flatpickr), so it doesn't make sense for us to add all of those inputs/outputs in case there is a change in flatpickr.

In such case, to resolve this, we would need to phase it out via breaking changes, so the reasonable thing to do on our end was allow for a user to pass in a flatpickr options object.

That being said, I definitely think we do need to **_improve_** our storybook documentation to better showcase this.